docs/projects/ad7606x_fmc/index.rst

```diff
@@ -0,0 +1,432 @@
+JP12               B                Connects the V2- line to the J6 SMB connector
+JP13               B                Connects the V3- line to the J7 SMB connector
+JP14               B                Connects the V4- line to the J8 SMB connector
+P13                Inserted         Connects the V5- line to ground
+P14                Inserted         Connects the V8- line to ground
+P15                Inserted         Connects the V6- line to ground
+P16                Inserted         Connects the V7- line to ground
+S1                 On               Controls the OS0, OS1, and OS2 pins
+================== ================ =============================================
+
+CPU/Memory interconnects addresses
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The addresses are dependent on the architecture of the FPGA, having an offset
+added to the base address from HDL (see more at :ref:`architecture`).
+
+========================  ===========
+Instance                  Address
+========================  ===========
+axi_ad7606x_dma           0x44A3_0000
+spi_clkgen                0x44A7_0000
+ad7606_pwm_gen            0x44B0_0000
+spi_ad7616_axi_regmap **  0x44A0_0000
+axi_ad7606x *             0x44A0_0000
+========================  ===========
+
+.. admonition:: Legend
+   :class: note
+
+   -   ``*`` instantiated only for INTF=0 (parallel interface)
+   -   ``**`` instantiated only for INTF=1 (serial interface)
+
+I2C connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 20 20 20 20 20
+   :header-rows: 1
+
+   * - I2C type
+     - I2C manager instance
+     - Alias
+     - Address
+     - I2C subordinate
+   * - PL
+     - iic_fmc
+     - axi_iic_fmc
+     - 0x4162_0000
+     - ---
+   * - PL
+     - iic_main
+     - axi_iic_main
+     - 0x4160_0000
+     - ---
+
+SPI connections
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 1
+
+   * - SPI type
+     - SPI manager instance
+     - SPI subordinate
+     - CS
+   * - PL
+     - axi_spi_engine
+     - AD7606
+     - 0
+
+GPIOs
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The Software GPIO number is calculated as follows:
+
+-  Zynq-7000: if PS7 is used, then offset is 54
+
+.. list-table::
+   :widths: 25 25 25 25
+   :header-rows: 2
+
+   * - GPIO signal
+     - Direction
+     - HDL GPIO EMIO
+     - Software GPIO
+   * -
+     - (from FPGA view)
+     -
+     - Zynq-7000
+   * - adc_serpar
+     - OUT
+     - 39
+     - 93
+   * - adc_refsel *
+     - OUT
+     - 38
+     - 92
+   * - adc_first_data **
+     - IN
+     - 38
+     - 92
+   * - adc_reset
+     - OUT
+     - 37
+     - 91
+   * - adc_stby
+     - OUT
+     - 36
+     - 90
+   * - adc_range
+     - OUT
+     - 35
+     - 89
+   * - adc_os
+     - OUT
+     - 34:32
```

Comment:
Tiny mistake: should be `-` instead of `:`

Reply:
Done.